design/ui/pagination/README.mdx

```diff
@@ -0,0 +1,170 @@
+---
```

Comment:
@lordpixel none of the comments on this file are things for you to change... they are merely me sharing some thoughts about how this will eventually work in terms of the MDX tooling so we have a more shared understanding of the direction at a technical level. I think that will be helpful in terms of what sorts of things we might need to come back and clean up as the tooling improves.

Reply:
got it!

Reply:
@CaffeinatedAllie these comments are also probably useful for you as well, since these docs, curating them, and maintaining them, need to serve both developers and designers.